src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const { type, extraData, keysToRemove } of actions) {
+    switch (type) {
+      case 'addProperties':
+        Object.assign(state, extraData);
+        break;
+
+      case 'removeProperties':
+        keysToRemove.forEach(key => {
+          delete state[key];
+        });
+        break;
+
+      case 'clear':
+        Object.keys(state).forEach(key => {
+          delete state[key];
+        });
+        break;
```

Comment:
But a default case should be present too. Just throw some error in it
![image](https://user-images.githubusercontent.com/96409563/220188025-46e782e8-d5f6-42c8-963a-28536e6c7ab9.png)